### ISSUE TITLE
Suggestion for improvement im switch menu

### ISSUE THREAD

**Issue by siegfriedhoellrigl1** [openwrt/luci#7303]
It would be very nice if we could have a "Comment" field for each port under :
Network - Switch menu.

**Comment by systemcrash** [comment]
What fields do you see today?

**Comment by siegfriedhoellrigl1** [comment]
Today I can only make comments for the VLANs.
Not for the physical Ports.

**Comment by dannil** [comment]
Which device are you running? The `Network` -> `Switch` menu isn't present on targets which has migrated to [DSA (Distributed Switch Architecture)](https://docs.kernel.org/networking/dsa/dsa.html), so if we want to have a place to make comments like this it should probably not be there.

**Comment by siegfriedhoellrigl1** [comment]
At the moment I am using 3 TP-Links :
Archer A7 v5
TL-WDR4300 v1
TL-WR1043ND v2.
So you think, this is useless, because the switch menu will appear in a short time, correct ?

**Comment by dannil** [comment]
> At the moment I am using 3 TP-Links : Archer A7 v5 TL-WDR4300 v1 TL-WR1043ND v2. So you think, this is useless, because the switch menu will appear in a short time, correct ?

Maybe not useless, some devices will probably never get DSA and stay on swconfig, so for them the switch menu would still appear. But, if we're thinking of a general solution about being able to add comments to the physical ports, there maybe could be a nice solution that would incorporate both swconfig and DSA targets.

**Comment by systemcrash** [comment]
Admittedly, it's harmless to have a `comment` field, since it's only ever used in the GUI here. Lower level scripts aren't looking for it. DSA stuff is under Interfaces, right?

**Comment by dannil** [comment]
> Admittedly, it's harmless to have a `comment` field, since it's only ever used in the GUI here. Lower level scripts aren't looking for it. DSA stuff is under Interfaces, right?

Yes, for example VLAN:s are configured as a new device under interfaces with DSA. If moving this to some "common" place (it would maybe even need a new UI) is too hard I'm fine with just adding a comment field as well.
